src/node.cc

```diff
@@ -2257,6 +2258,22 @@ inline napi_addon_register_func GetNapiInitializerCallback(DLib* dlib) {
       reinterpret_cast<napi_addon_register_func>(dlib->GetSymbolAddress(name));
 }
 
+// Initialize the addon, and close the DLib if this is a second-or-later
+// initialization call.
```

Comment:
We do not close the addon. We close the `dlopen()` handle, because, if the symbol was already in the cache, we know that we've obtainted a `dlopen()` handle to it in the past, and thus we can conclude that the `dlopen()` handle we received this time is not a new handle, the closing of which would cause the addon to be unmapped from our address space, but rather a refcounted version of the old handle we received in the past. Thus, closing the new handle is safe.